src/app/[locale]/campagne-partenaire/[pollSlug]/page.tsx

```diff
@@ -0,0 +1,44 @@
+import FAQ from '@/components/landing-pages/FAQ'
+import Partners from '@/components/landing-pages/Partners'
+import Markdown from '@/design-system/utils/Markdown'
+import type { Locale } from '@/i18nConfig'
+import { fetchPartnerCampaign } from '@/services/cms/fetchPartnerCampaign'
+import { redirect } from 'next/navigation'
+import PartnerCampaignContent from './_components/PartnerCampaignContent'
+
+export default async function PartnerCampaignPage({
+  params,
+}: {
+  params: Promise<{ pollSlug: string; locale: Locale }>
+}) {
+  const { locale, pollSlug } = await params
+
+  const partnerCampaign = await fetchPartnerCampaign({
+    locale,
+    pollSlug,
+  })
+
+  if (!partnerCampaign) {
+    redirect('/404')
+  }
+
+  return (
+    <PartnerCampaignContent
+      pollSlug={pollSlug}
+      partnerCampaign={partnerCampaign}
+      partnersComponent={<Partners locale={locale} />}
+      faqComponent={
+        partnerCampaign.questions?.length && (
```

Comment:
Must be converted to a boolean no ?

Reply:
Why ? Could it display 0 if length is equal to 0 ?